From brice@myri.com  Tue Aug 14 15:26:01 2007
From: Brice Goglin <brice@myri.com>
Date: Tue, 14 Aug 2007 12:43:48 +0200
Subject: PCI: make pcie_get_readrq visible in pci.h
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Stephen Hemminger <shemminger@linux-foundation.org>, Peter Oruba <peter.oruba@amd.com>, linux-pci@atrey.karlin.mff.cuni.cz
Message-ID: <46C18764.2010204@myri.com>


[PATCH] PCI: make pcie_get_readrq visible in pci.h

pcie_get_readrq() is EXPORT_SYMBOL'ed, but its prototype is not
visible in pci.h, add it there.

This is needed by some network drivers.

Signed-off-by: Brice Goglin <brice@myri.com>
Acked-by: Peter Oruba <peter.oruba@amd.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/pci.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -557,6 +557,7 @@ int pci_set_consistent_dma_mask(struct p
 int pcix_get_max_mmrbc(struct pci_dev *dev);
 int pcix_get_mmrbc(struct pci_dev *dev);
 int pcix_set_mmrbc(struct pci_dev *dev, int mmrbc);
+int pcie_get_readrq(struct pci_dev *dev);
 int pcie_set_readrq(struct pci_dev *dev, int rq);
 void pci_update_resource(struct pci_dev *dev, struct resource *res, int resno);
 int __must_check pci_assign_resource(struct pci_dev *dev, int i);
